man/runc-features.8.md

```diff
@@ -0,0 +1,28 @@
+% runc-features "8"
+
+# NAME
+**runc-features** - show implemented features
+
+# SYNOPSIS
+**runc features**
+
+# DESCRIPTION
+The **features** command shows the implemented features in JSON format. Features are properties of runc, such as the minimum and maximum accepted OCI versions. The implemented features may not always be available, depending on the kernel version, system libraries, CPU architecture, etc. For more information about features, you can check the features section in the runtime-spec on GitHub.
```

Comment:
1. nit: It would be better to limit the width to 79 characters or so.
2. Add a link to runtime-spec features doc here and/or to SEE ALSO section.